src/relative-time/properties.js

```diff
@@ -0,0 +1,51 @@
+define(function() {
+
+/**
+ * properties( unit, cldr, options )
+ *
+ * @unit [String] eg. "day", "week", "month", etc.
+ *
+ * @cldr [Cldr instance].
+ *
+ * @options [Object]
+ * - form: [String] eg. "short" or "narrow". Or falsy for default long form
+ * - minWordOffset [Optional Number] The maximum offset when special offset words like
+ *  yesterday and tomorrow will be looked for. Some languages provide several of these.
+ *  default null -> use all available
+ *  Set to 0 to not use any except today, now etc.
+ *
+ * Return relative time properties.
+ */
+return function( unit, cldr, options ) {
+
+	options = options || {};
```

Comment:
Let's remove this, since `.relativeTimeFormatter()` has already performed `options = options || {}`.